examples/tv-app/android/java/AppImpl.cpp

```diff
@@ -45,6 +45,7 @@
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/DeviceInstanceInfoProvider.h>
 
+using namespace app::Clusters;
```

Comment:
I would have expected this to be `using namespace chip::app::Clusters;` and probably come after the `using namespace chip;`.